Source/WebCore/html/canvas/CanvasRenderingContext2DBase.h

```diff
@@ -345,10 +345,14 @@ class CanvasRenderingContext2DBase : public CanvasRenderingContext, public Canva
         ApplyPostProcessing = 1 << 3,
         PreserveCachedImageData = 1 << 4,
     };
-    void didDraw(std::optional<FloatRect>, OptionSet<DidDrawOption> = { DidDrawOption::ApplyTransform, DidDrawOption::ApplyShadow, DidDrawOption::ApplyClip, DidDrawOption::ApplyPostProcessing });
-    void didDrawEntireCanvas();
-    void didDraw(bool entireCanvas, const FloatRect&);
-    template<typename RectProvider> void didDraw(bool entireCanvas, RectProvider);
+
+    static OptionSet<DidDrawOption> defaultDidDrawOptions();
```

Comment:
This and the next line down can use `static constexpr OptionSet<>` declared here in the header.